internal/adapter/in/controller/account_controller_test.go

```
@@ -83,18 +83,20 @@ var _ = Describe("Account Controller", func() {
 					Name: accountOperatorNamespace,
 				},
 			}
-			k8sClient.Create(ctx, ns)
+			err := k8sClient.Create(ctx, ns)
```

The tests are failing due to that the namespace already exists.
Ignore any errors during namespace creation.